packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -89,14 +89,33 @@ class MessageEventBus extends EventEmitter {
 		// - cycle event logs and start the logging to a fresh file
 		// - retry sending events
 		LoggerProxy.debug('Checking for unsent event messages');
-		const unsentMessages = await this.getEventsUnsent();
+		const unsentAndUnfinished = await this.getUnsentAndUnfinishedExecutions();
 		LoggerProxy.debug(
 			`Start logging into ${
 				(await this.logWriter?.getThread()?.getLogFileName()) ?? 'unknown filename'
 			} `,
 		);
 		await this.logWriter?.startLogging();
-		await this.send(unsentMessages);
+		await this.send(unsentAndUnfinished.unsentMessages);
+
+		if (unsentAndUnfinished.unfinishedExecutions.size > 0) {
+			for (const executionId of unsentAndUnfinished.unfinishedExecutions) {
+				LoggerProxy.debug(`Found unfinished execution ${executionId} in event log(s)`);
+				// TODO: commented out for now, until it has been decided how to handle unfinished executions
```

Comment:
I think it's worth explaining either here, the PR or in the ticket why this is being commented.

Reply:
This is part of an upcoming PR that deals with handling workflows that, in the frontend, show up as 'Status Unknown', usually because they crashed. Tbh. it really does not have to say in here for this PR and I'll remove it...